engine/apps/public_api/serializers/integrations.py

```diff
@@ -8,14 +8,53 @@
 from apps.base.messaging import get_messaging_backends
 from common.api_helpers.custom_fields import TeamPrimaryKeyRelatedField
 from common.api_helpers.exceptions import BadRequest
-from common.api_helpers.mixins import NOTIFICATION_CHANNEL_OPTIONS, EagerLoadingMixin
+from common.api_helpers.mixins import PHONE_CALL, SLACK, SMS, TELEGRAM, WEB, EagerLoadingMixin
 from common.jinja_templater import jinja_template_env
 from common.utils import timed_lru_cache
 
 from .integtration_heartbeat import IntegrationHeartBeatSerializer
 from .maintenance import MaintainableObjectSerializerMixin
 from .routes import DefaultChannelFilterSerializer
 
+# Behaviour templates are named differently in public api
+PUBLIC_BEHAVIOUR_TEMPLATES_FIELDS = ["resolve_signal", "grouping_key", "acknowledge_signal", "source_link"]
+
+# TEMPLATE_PUBLIC_API_NAME_TO_DB_FIELD is map from template name in public api to its db field.
+TEMPLATE_PUBLIC_API_NAME_TO_DB_FIELD = {
```

Comment:
Please add a comment that `TEMPLATE_PUBLIC_API_NAME_TO_DB_FIELD` is only applied for legacy messengers